zkevm-circuits/src/tx_circuit.rs

```diff
@@ -330,6 +353,20 @@ impl<F: Field> SubCircuitConfig<F> for TxCircuitConfig<F> {
         let is_access_list_storage_key = meta.advice_column();
         let field_rlc = meta.advice_column();
 
+        // Chunk bytes accumulator
+        let is_chunk_bytes = meta.advice_column();
+        let chunk_bytes_len = meta.advice_column();
+        let chunk_txbytes_rlc = meta.advice_column_in(SecondPhase);
+        let chunk_txbytes_hash = meta.advice_column_in(SecondPhase);
+        let chunk_txbytes_len_acc = meta.advice_column();
+        let pow_of_rand = meta.advice_column_in(SecondPhase);
+
+        meta.enable_equality(chunk_bytes_len);
+        meta.enable_equality(chunk_txbytes_rlc);
```

Comment:
why do we need to put these 4 columns except `chunk_txbytes_hash` into permutation argument? It seems only the last tx's `chunk_txbytes_hash` cell needs to be copied to pi circuit.